corehq/apps/consumer_user/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from .views import login_view, register_view, success_view, logout_view, change_password_view, \
+    domains_and_cases_list_view, change_contact_details_view
+
+app_name = 'consumer_user'
+
+urlpatterns = [
+    path('signup/<invitation>/', register_view, name='patient_register'),
```

Comment:
I think it's a bit confusing that you use the word `patient` here since we are calling these `consumer_user`s

Reply:
Changed patient to consumer_user at all places its being used